tools/getProof.js

```diff
@@ -0,0 +1,44 @@
+const debug = require('debug')('getProof');
+const { helpers, Output, Outpoint, Tx } = require('leap-core');
+const { periodOfTheBlock } = require('./helpers/helpers');
+const { bufferToHex } = require('ethereumjs-util');
+const Web3 = require('web3');
+
+const web3 = new Web3('http://node1.testnet.leapdao.org:8645');
```

Comment:
Switch to https load balancer? :)

Reply:
i don't know that code & not needed, slipped in during rebase. removed.